test/Github/Tests/Api/IntegrationTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Github\Tests\Api;
+
+class IntegrationTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function shouldFindRepositoriesForIntegration()
+    {
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('/integration/installations');
+
+        $api->findInstallations();
+    }
+
+    /**
+     * @test
+     */
+    public function shouldGetRepositoriesFromInstallation()
+    {
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('/installation/repositories');
+
+        $api->listRepositories('1234');
```

Comment:
Test that the parameter is used. Also make a second test where you not using any $userId